README.md

```diff
@@ -235,6 +236,13 @@ for result in cursor {
 }
 ```
 
+## Web Frameworks
```

Comment:
Might want to include the word "Example" or "Examples" in the heading to quickly grab the attention of people scanning for it.